eclair-core/src/main/scala/fr/acinq/eclair/channel/Helpers.scala

```diff
@@ -698,7 +698,7 @@ object Helpers {
         val closingTx = Transactions.makeClosingTx(commitInput, actualLocalScript, actualRemoteScript, localParams.isInitiator, dustLimit, closingFees.preferred, localCommit.spec)
         val localClosingSig = keyManager.sign(closingTx, keyManager.fundingPublicKey(commitments.localParams.fundingKeyPath), TxOwner.Local, commitmentFormat)
         val closingSigned = ClosingSigned(channelId, closingFees.preferred, localClosingSig, TlvStream(ClosingSignedTlv.FeeRange(closingFees.min, closingFees.max)))
-        log.info(s"signed closing txid=${closingTx.tx.txid} with closing fee=${closingSigned.feeSatoshis}")
+        log.debug(s"signed closing txid=${closingTx.tx.txid} with closing fee=${closingSigned.feeSatoshis}")
```

Comment:
It was nice to have the fee, although this does feel like debug-level info. Well...

Reply:
But we actually already have it logged at two places in `info` already:

- in the `closing_signed` message that's logged entirely
- in the handler for `closing_signed` in `Channel.scala` in the branches where we finalize the closing tx

That's why I moved it to `debug`, it's really redundant with other logs we're keeping.